README.md

```diff
@@ -86,6 +86,8 @@ If you want to use typing, you can hit `alt+t`, even though it was started with
 
 Finally, with `--action copy` (or `-a copy`) you can also tell `rofimoji` to only copy the selected characters to your clipboard.
 
+You can specify multiple actions by providing a space-separated list. For example, `--action copy type` would both type the character and copy it to your clipboard.
```

Comment:
Not sure this belongs here... it's not completely wrong, but it also doesn't describe the insertion methods.
So, do you think it's enough to describe it above, in the options table?